doc/tutorials/SIR-X.ipynb

```diff
@@ -13,7 +13,7 @@
    "source": [
     "This notebook exemplifies how Open-SIR can be used to fit the SIR-X model by [Maier and Dirk (2020)](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full) to existing data and make predictions. The SIR-X model is an standard generalization of the Susceptible-Infectious-Removed (SIR) model, which includes the influence of exogeneous factors such as policy changes, lockdown of the whole population and quarantine of the infectious individuals.\n",
     "\n",
-    "To validate the Open-SIR implementation of the SIR-X model, it will be attempted to reproduce the parameter fitting published in the [suplementary material](https://science.sciencemag.org/cgi/content/full/science.abb4557/DC1) of the original recent article published by [Maier and Dirk (2020)](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full). For simplicity, the validation will be performed only for the city of Guangdong, China."
+    "To validate the Open-SIR implementation of the SIR-X model, it will be attempted to reproduce the parameter fitting published in the [supplementary material](https://science.sciencemag.org/cgi/content/full/science.abb4557/DC1) of the original recent article published by [Maier and Dirk (2020)](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full). For simplicity, the validation will be performed only for the city of Guangdong, China."
```

Comment:
"It will be attempted" sounds "off". Should it say something like "we will attempt", or "it will attempt"?

What do you think @jia200x ?

Reply:
I would not use "we" because it is ambiguous. We the authors? we the universe? we the reader and I?

I agree that the syntax is weird and I will find something better.